packages/react-native-room-kit/src/components/Preview.tsx

```diff
@@ -88,6 +95,13 @@ export const Preview = ({
     setupAudioVideoOnPreview().then((r) => console.log(r));
   }, []);
 
+  const canPublishAudio = useCanPublishAudio();
+  const isLocalAudioMuted = useSelector(
+    (state: RootState) => state.hmsStates.isLocalAudioMuted
+  );
+
+  const showNoiseCancellationButton = canPublishAudio && !isLocalAudioMuted;
```

Comment:
@ygit we should also check if noise cancellation is enabled for room

Reply:
Added the check